modules/local/build_intervals/main.nf

```diff
@@ -8,7 +8,7 @@ process BUILD_INTERVALS {
         'quay.io/biocontainers/gawk:5.1.0' }"
 
     input:
-    path fasta_fai
+    tuple val(meta), path(fasta_fai)
```

Comment:
I'll remove the meta out of the fasta_fai channel with a map instead of adding meta here, we use fasta_fai a lot in other downstream modules/subworkflows